dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -0,0 +1,291 @@
+#!/bin/bash
+
+set -e
+
+init() {
+#OS specific defaults
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    DEFAULT_OC_PUBLIC_HOSTNAME="192.168.65.2"
+    DEFAULT_OC_PUBLIC_IP="192.168.65.2"
+    [ -z "$CHE_OAUTH_GITHUB_CLIENTID" ] && export CHE_OAUTH_GITHUB_CLIENTID=d210c15f1a0c026a808c;
```

Comment:
secrets shouldn't be in a script, no ?

Reply:
that is secrets for default mac OS IP, I can remove it but this way it is much easier to use it on mac OS

Reply:
But I agree that this is not a good thing and I will remove it

Reply:
I'm fine as long it's secret not so secret ;-)

Reply:
removed :)